### PR TITLE
Refactor duplicate code in CQLResultSetKeyIteratorTest

### PR DESCRIPTION
**Pull request by porunov**
Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java [hunk 1]
```diff
@@ -92,29 +92,9 @@ public void testEmpty() throws IOException {
...
-        final Function1<Integer, ByteBuffer> randomLong = idx -> {
-            final ByteBuffer buffer = ByteBuffer.allocate(Long.BYTES).putLong(random.nextLong());
-            buffer.flip();
-            return buffer;
-        };
-
-        final Array<Tuple2<ByteBuffer, Array<Tuple2<ByteBuffer, ByteBuffer>>>> keysMap = Array.range(0, random.nextInt(100) + 100)
-                .map(randomLong)
-                .map(key -> Tuple.of(key, Array.rangeClosed(0, random.nextInt(100) + 1)
-                        .map(idx -> Tuple.of(randomLong.apply(idx), randomLong.apply(idx)))));
```
Comment (chupman) [inline janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java]: please remove empty line.

File: janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java [hunk 2]
```diff
@@ -143,29 +123,11 @@ public void testUneven() throws IOException {
...
```
Comment (chupman) [inline janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java]: please remove empty line

File: janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java [hunk 3]
```diff
@@ -143,29 +123,11 @@ public void testUneven() throws IOException {
...
 
-        final Seq<Row> rows = keysMap.flatMap(tuple -> tuple._2.map(columnAndValue -> {
-            final Row row = mock(Row.class);
-            when(row.getBytes("key")).thenReturn(tuple._1);
-            when(row.getBytes("column1")).thenReturn(columnAndValue._1);
-            when(row.getBytes("value")).thenReturn(columnAndValue._2);
-            return row;
-        }));
+        final Random random = new Random();
```
Comment (chupman) [inline janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java]: please remove empty line

File: janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java [hunk 4]
```diff
@@ -199,6 +161,21 @@ public void testPartialIterateColumns() throws IOException {
 
     @Test
     public void testNoIterateColumns() throws IOException {
+
```
Comment (chupman) [inline janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLResultSetKeyIteratorTest.java]: please remove empty line

**Comment by porunov** [comment]
@chupman I've removed those empty lines.

**Comment by porunov** [comment]
@FlorianHockmann , @chupman I believe this PR can be merged.

**Pull request merged**